python/cugraph/cugraph/structure/property_graph.py

```diff
@@ -1062,6 +1062,35 @@ def get_edge_data(self, edge_ids=None, types=None, columns=None):
 
         return None
 
+    def fillna_vertices(self, val=0):
+        """
+        Fills empty vertex property values with the given value, zero by default.
+        Fills in-place.
+
+        Parameters
+        ----------
+        val : object, cudf.Series, or dict
```

Comment:
Small nit... should we just say `Series` instead of `cudf.Series`? Whether it accepts a cudf Series or pandas Series depends on user input data.

Reply:
you're right, sorry